app/faucet/views.py

```diff
@@ -90,12 +92,12 @@ def save_faucet(request):
         }, status=400)
     fr = FaucetRequest.objects.create(
         fulfilled=False,
-        github_username=github_profile,
-        input_github_username=profile_handle,
+        github_username=request.user.username,
```

Comment:
Not in this case. We're validating that `profile` is present on L67 which relies on L65 that checks if `request.user.is_authenticated`, otherwise the user will receive a 401.